geometry_msgs/Point gaze_point_3d
geometry_msgs/Point norm_pos
geometry_msgs/Point gaze_normals_3d_left
geometry_msgs/Point gaze_normals_3d_right
geometry_msgs/Point eye_centers_3d_left
geometry_msgs/Point eye_centers_3d_right
float64 pupil_confidence
float64 calib_rotation
uint32 label
uint32 header
string surface_name
bool gaze_on_surf
geometry_msgs/Point s_norm_gp

